simularium_metrics_calculator/bin/example.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from typing import Any, Dict, List
+
+from simulariumio import InputFileData
+
+from simularium_metrics_calculator import (
+    PLOT_TYPE,
+    SCATTER_PLOT_MODE,
+    MetricsManager,
+    PlotInfo,
+)
+
+
+def log_available_metrics(metrics: List[Dict[str, Any]]) -> None:
+    print("\nAvailable metrics:")
+    for index, metric in enumerate(metrics):
+        uid = metric["uid"]
+        display_name = metric["display_name"]
+        metric_type = metric["metric_type"]
+        print(f"  {index} : {display_name}, uid = {uid} ({metric_type})")
+    print("")
+
+
+def main() -> None:
+    # create main class (MetricsManager) with a simularium trajectory
+    manager = MetricsManager(
+        input_data=InputFileData(
+            file_path=(
+                "simularium_metrics_calculator/tests/data/"
+                "aster_pull3D_couples_actin_solid_3_frames_small.json"
+            )
+        ),
+    )
```

Comment:
this is looking nice and clean to me but I just wonder about one thing.  As a server/service, does it possibly make sense to have one MetricsManager that lives for the lifetime of the service and can serve back metrics for many different InputFileDatas?  
It's hard for me to tell that the sequence of operations would be the same when the processing is initiated by remote network messages.
Also, the use of the word Manager is sometimes considered a no-no because it can get over used in larger code bases and it's not specific enough.  Not super important yet, but consider `Metrics`, `MetricsService`, `PlotDataService`

Reply:
If there was one `MetricsManager` for multiple trajectories, I think there could eventually be issues with keeping them all in memory, and maybe it's better to have the user decide how to handle that? It seems like it would depend on the use case. But one also does make sense in order to not have redundant metric registry dicts around.